gtkctree.h and gtkclist.h include each other, the order of inclusion
is important in gtk-2.13.3
https://bugs.launchpad.net/inkscape/+bug/246868

Lubomir Rintel <lkundrak@v3.sk>

--- inkscape-0.46+devel.orig/src/widgets/sp-xmlview-attr-list.h	2008-06-24 13:38:07.000000000 +0200
+++ inkscape-0.46+devel/src/widgets/sp-xmlview-attr-list.h	2008-07-05 13:03:50.000000000 +0200
@@ -13,6 +13,7 @@
  */
 
 #include <stdio.h>
+#include <gtk/gtkctree.h>
 #include <gtk/gtkclist.h>
 #include "../xml/repr.h"
 
